apps/server/src/utils/coerceType.ts

```diff
@@ -44,6 +44,16 @@ export function coerceBoolean(value: unknown): boolean {
   return Boolean(value);
 }
 
+export function coerceBooleanLoose(value: unknown, trueOption?: string): boolean {
+  if (value === undefined || value === null) {
+    return false;
+  }
+  if (value === trueOption) {
```

Comment:
Not sure about this true option. 

What is the Boolean value that we get from excel and Google sheets??

I think we should check for that Boolean instead. And if the input is a string, any non empty string is a TRUE

Reply:
we might get 'TRUE' or 'FALSE' as text withc is handled by the normal `coerceBoolean`
and in the pareser code we had an extra allowens for just an 'x'
I have also noticed a `null` comming through sometimes

Reply:
but then it sounds like coerceBoolean does everything we need?
How about something like 
```ts
function parseBooleanString(maybeText: string | null) {
  // falsy values would be nullish or empty string
  if (!maybeText) {
    return false;
  }
  return maybeText.toLowerCase() !== 'false';
}

```